funnel/templates/proposals.html.jinja2

```diff
@@ -5,9 +5,40 @@
 {% block description %}{{ project.title }}{% if g.profile %} &ndash; {{ g.profile.description|striptags }}{% endif %}{% endblock %}
 
 {%- block pageheaders %}
-  {% assets "css_leaflet" -%}
-    <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}" />
-  {%- endassets -%}
+  <script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Event",
```

Comment:
Event schema should be in the project page and maybe the schedule page. It's not required for the proposals page.